README.md

```diff
@@ -1,13 +1,6 @@
 # rapid-response-xblock
 A django app plugin for edx-platform
 
-__NOTE:__ We had to make several fixes to XBlock Asides in `edx-platform` in order to get rapid response working. 
-The `edx-platform` branch/tag you're using must include these commits for rapid response to work:
-
-- https://github.com/mitodl/edx-platform/commit/b26db017a55140bb7940c3fbfac5b4f27128bffd
-- https://github.com/mitodl/edx-platform/commit/96578f832d786d90162c555f1cfa08f69ba294d2
-- https://github.com/mitodl/edx-platform/commit/1bd36be3b31210faa8af09fc28ff4a885807e20e
```

Comment:
We used to cherry-pick some commits for the rapid response xBlock to work properly but that was before those changes were part of Open edX. Later on, these commits were [merged upstream](https://github.com/openedx/edx-platform/pull/19089/files) but we probably never reflected that into readme. So I'm removing this because we don't need these cherry-picks now.